example_configs/vaultwarden.md

```diff
@@ -0,0 +1,32 @@
+# Configuration for Vaultwarden
+
+https://github.com/ViViDboarder/vaultwarden_ldap will send an invitation to any member of the group ```vaultwarden```.
```

Comment:
Nit: For inline code segments, can you use a single backtick? And triple for big blocks, with optionally the syntax at the beginning:

````
```toml
vaultwarden_url = "http://bitwarden"
```
````